### DISCUSSION TITLE
MCP2515 and my_user_config.h

### DISCUSSION THREAD

**Discussion by javier6d2** [arendst/Tasmota#15136] [Q&A] [answered]
Hi, may be a silly question, I want to compile my own tasmota build with xsns_87_mcp2515.ino driver. I think I must compile my build adding the following to user_config_override.h:

```
#define USE_SPI
#define USE_MCP2515  
```
But maybe I'm wrong because I don´t find `#define USE_MCP2515`  [in my_user_config.h](https://github.com/arendst/Tasmota/blob/development/tasmota/my_user_config.h) file. 

Any help is appreciated

Kind regards

**Comment by barbudor** [comment, marked as answer]
What is in `my_user_config.h` is mostly to be taken as exemples. It is not necesseraly fully exhaustive. Would be better if it is but sometime the author of a driver just forget about adding it here...

However, the 2 defines that you listed are exactly what you need to add in your `user_configçoverride.h`

**Comment by javier6d2** [comment]
Very well, I had thought about that but wasn't sure.
Thank you very much for the clarification.
